### PR TITLE
Use `node/res/bp_parachain.json` specification for parachain deployment

### PR DESCRIPTION
**Pull request by c410-f3r**
Hopefully we won't have more genesis conflicts

### CODE REVIEW

File: node/res/bp_parachain.json [hunk 1]
```diff
@@ -2,9 +2,7 @@
   "name": "Zeitgeist Battery Park",
   "id": "battery_park",
   "chainType": "Live",
-  "bootNodes": [
-    "/ip4/139.162.171.58/tcp/30333/p2p/12D3KooWPvu5rpH2FNYnAmiQ8X8XqkMiuSFTjH2jwMCSjoam7RGQ"
-  ],
```
Comment (c410-f3r) [inline node/res/bp_parachain.json:7]: This bootnode is for the standalone node

File: .github/workflows/docker-hub-parachain.yml [hunk 2]
```diff
@@ -11,6 +11,15 @@ jobs:
...
         uses: actions/checkout@v2
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            zeitgeistpm/zeitgeist-node-parachain
+          tags: |
+            type=sha
```
Comment (c410-f3r) [inline .github/workflows/docker-hub-parachain.yml:21]: Will deploy a new image based on the commit sha hash

**Review by lsaether — APPROVED** [review]

File: .github/workflows/docker-hub-parachain.yml (hunk 2 above)
Comment (lsaether) [reply .github/workflows/docker-hub-parachain.yml:21]: Does it also push to latest?

Comment (c410-f3r) [reply .github/workflows/docker-hub-parachain.yml]: It will now. Thanks

**Pull request merged**